clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumptionTask.java

```diff
@@ -50,6 +52,15 @@ class ConsumptionTask implements Runnable {
   private final AggKafkaConsumerServiceStats aggStats;
   private final ConsumerSubscriptionCleaner cleaner;
 
+  /**
+   * Maintain rate counter with default window size to calculate the message and bytes rate at topic partition level.
+   */
+  private final Map<PubSubTopicPartition, Rate> messageRatePerTopicPartition = new HashMap<>();
```

Comment:
nit: Do you think it is better to use VeniceConcurrentHashMap here? As I am not sure if there are more than one thread updating the map. Even it is not, I feel like VeniceConcurrentHashMap is always better than plain HashMap.

Reply:
Yeah, we should change it to VeniceConcurrentHashMap here. As there is one thread write: ConsumtpionTask one thread read from this hashmap.